lib/lightstep/span.rb

```diff
@@ -78,6 +78,11 @@ def set_baggage(baggage = {})
       )
     end
 
+    # Returns the baggage associated with the span
+    def baggage
```

Comment:
this should be accessed via the `SpanContext` per https://github.com/opentracing/opentracing-ruby/blob/master/lib/opentracing/span_context.rb#L6